src/style.scss

```diff
@@ -15,26 +15,43 @@ $icon-font-path: '../node_modules/bootstrap-sass/assets/fonts/bootstrap/';
   left: initial;
   bottom: 10px;
   right: 10px;
+}
+
+@mixin ol-control-item {
+  color: $vl-text-color;
+  background: white;
+  width: 3.5rem;
+  height: 3.5rem;
+  margin: 0;
+  border-radius: 0;
+
+  &:hover {
+    background-color: white;
+    cursor: pointer;
+  }
+
+  &:focus {
+    outline: 0;
+    border: 0.1rem solid #ffc515;
+    box-shadow: inset 0 0 0 0.1rem #ffc515;
+    background-color: white;
+  }
+}
+
+.ol-zoom {
+  @include ol-control;
   
   .ol-zoom-in, .ol-zoom-out {
-    color: $vl-text-color;
-    background: white;
-    width: 3.5rem;
-    height: 3.5rem;
-    margin: 0;
-    border-radius: 0;
+    @include ol-control-item;
+  }
+}
 
-    &:hover {
-      background-color: white;
-      cursor: pointer;
-    }
+.ol-full-screen {
+  @include ol-control;
+  bottom: 92px;
 
-    &:focus {
-      outline: 0;
-      border: 0.1rem solid #ffc515;
-      box-shadow: inset 0 0 0 0.1rem #ffc515;
-      background-color: white;
-    }
+  .ol-full-screen-false, .ol-full-screen-true {
```

Comment:
Dit is voor als het af en aan staat? Misschien iets duidelijker maken dan: ol-full-screen-activated of ...-active / deactivated?

Reply:
+1

Reply:
Ah te snel geweest, dat is openlayers specifieke variabele waar wij niet veel over te zeggen hebben wellicht?

Reply:
Dat zijn inderdaad OL classes waar ik gebruik van maak om de stijl wat aan te passen.